### ISSUE TITLE
Not working, game not starting

### ISSUE THREAD

**Issue by kilyan82** [rebtd7/FF13Fix#18]
Hello, i've put the dll and and the ini file in the STEAM\steamapps\common\FINAL FANTASY XIII\white_data\prog\win\bin directory, but when i click on play button on steam, the launcher appears and if i click start nothing happens. If i delete the dll, the game is booting, so the issue must be in this fix somehow. I have the double pack version of the game, ffxiii+ffxiii-2, but i have installed only ffxiii, so shouldn't be an issue.
[FF13Fix.log](https://github.com/rebtd7/FF13Fix/files/4458753/FF13Fix.log)

**Comment by kilyan82** [comment]
[FF13Fix.log](https://github.com/rebtd7/FF13Fix/files/4458738/FF13Fix.log)

**Comment by Nucleoprotein** [comment]
23:33:54.449	00018904	CreateDevice fail with HRESULT: 8876086C
Hymm, D3DERR_INVALIDCALL, try updating you GPU driver.

**Comment by kilyan82** [comment]
> 
> 
> `23:33:54.449 00018904 CreateDevice fail with HRESULT: 8876086C`
> Hymm, D3DERR_INVALIDCALL, try updating you GPU driver.
> 
> Also i see:
> 
> ```
> 23:33:50.723	00018904	HWND 0x00800C52: ClassName "CicMarshalWndClass", WindowName: "CicMarshalWnd"
> 23:33:50.724	00018904	HWND 0x00D108CE: ClassName "RTSSWndClass", WindowName: ""
> ```
> 
> You have somekind of OSD like RTSS - Rivatuner Statistics Server (bundled with MSI AfterBurner)? Disable it.

Unfortunately nothing worked, reinstalled drivers, they were already latest one, i disabled msi afterburner+riva tuner, i reinstalled dx jun 2010, becaise i have this error 
[![Screenshot from Gyazo](https://gyazo.com/3862657aa50774fa8d3b5b583bf58ebe/raw)](https://gyazo.com/3862657aa50774fa8d3b5b583bf58ebe)
One thing i did't mentioned i installed the ffxiii hd project mod and model swap.
Are they compatible with ffxiii fix?

**Comment by Nucleoprotein** [comment]
Please give us you PC specs, GPU, CPU model ect.

**Comment by kilyan82** [comment]
> 
> Please give us you PC specs, GPU, CPU model ect.
> Also you have modified ffxiiiimg.exe executable?

**Comment by kilyan82** [comment]
> 
> 
> > Please give us you PC specs, GPU, CPU model ect.
> > Also you have modified ffxiiiimg.exe executable?

GPU: NVIDIA GTX 1080
CPU: RYZEN 7 2700X
RAM:16GB DDR4 3200 MHZ FLARE-X CL 14
PSU: ANTEC EDGE 650W
MOBO; X470 TAICHI

I don't think ffxiii.img is modified, it works without ffxiii fix

**Comment by seuvah** [comment]
Do you run some kind of overlay, not steam

**Comment by kilyan82** [comment]
> 
> 
> Do you run some kind of overlay, not steam. Discord overlay or such give that error
> 
> Also this https://winaero.com/blog/disable-fullscreen-optimizations-windows-10/
> 
> https://www.groovypost.com/howto/run-old-software-windows-10-compatibility-mode/
> 
> Hope it helps.

Nope, still error and i disabled nearly everything
Here is how is my bin folder

[![Screenshot from Gyazo](https://gyazo.com/5aa30a101e4a05a443154e65b9e59f11/raw)](https://gyazo.com/5aa30a101e4a05a443154e65b9e59f11)

[FF13Fix.log](https://github.com/rebtd7/FF13Fix/files/4463496/FF13Fix.log)

**Comment by kilyan82** [comment]
i cleaned my registry and now it works... no clue
[FF13Fix.log](https://github.com/rebtd7/FF13Fix/files/4463571/FF13Fix.log)

**Comment by Gitii** [comment]
I just had the same crash.
`CreateDevice fail with HRESULT: 8876086C`.

I managed to start the game by avoiding the launcher and using the command line instead:
`"ffxiiiimg.exe" -Width=2560 -Height=1440 -VoiceJPMode -Shadow=4096 -MSAA=4`

If I start the game though the launcher with the same settings as above but with `-FullScreenMode=Force`,
it crashes. Without `-FullScreenMode=Force` it starts just fine.

If someone has the same problems:
I created an link to the executable, set the command line arguments and tweaked the ff13fix-settings (borderless, hideCursor).

Works great.
